Translation/Config.php

```diff
@@ -86,6 +86,11 @@
     private $keepOldMessages;
 
     /**
+     * @var bool
+     */
+    private $keepOldTranslationsMessages;
+
```

Comment:
Very minor nitpick, this variable should be named keepOldTranslationMessages (without the s on Translation)